lib/shelter.rb

```diff
@@ -1,2 +1,20 @@
 class Shelter
+
+  attr_reader :name, :capacity, :pets
+
+  def initialize(name,capacity,pets=[])
+    @name = name
+    @capacity = capacity
+    @pets = pets
+  end
+
+
+  def add_pet(pet)
+    @pets.push(pet)
```

Comment:
This works, and you will often see ruby-ists use the shovel operator as a shorter way to accomplish the same goal.

`@pets << pet`